### ISSUE TITLE
Privacy Badger interferes with reading.am add-on

### ISSUE THREAD

**Issue by tilgovi** [EFForg/privacybadgerfirefox#488]
The reading.am domain is blocked, but I purposely execute this code every time I want to use their add-on to log my reading.

**Comment by cooperq** [comment]
if you click on the settings button and move the slider for reading.am from red to yellow or green it should fix your issue.

**Comment by tilgovi** [comment]
Right. I wasn't sure if I should report these or not, as I thought there was a whitelist maintained here. What would you prefer?

**Comment by cooperq** [comment]
Oh definitely it's good to let us know so that we can potentially add it to the cookie block list. I was just giving you a workaround in cause you hadn't already done so yourself, getting updates from the cookie block list can take a while. Thanks for the report!
